mrbeam_ledstrips/tests/test_manually.py

```diff
@@ -0,0 +1,89 @@
+from __future__ import absolute_import
+import pytest
+from itertools import chain
+from os import path
+import random
+from six.moves import input
+from time import sleep
+
+from mrbeam_ledstrips.state_animations import LEDs, Colors, COMMANDS, COMMANDS_REQ_ARG
+from mrbeam_ledstrips.server import get_config
+
+from . import LEDLoopTester
+
+ANIMATION_TIME = .5
```

Comment:
good practice is to mention in the constant name the unit of time used, ms, ns, sec, min, ....
I suggest : `ANIMATION_TIME_IN_SEC`